### PR TITLE
Update to 1.9.4

### PR DESCRIPTION
**Pull request by Lordmau5**

### CODE REVIEW

File: src/main/java/com/superzanti/serversync/ServerSetup.java [hunk 1]
```diff
@@ -72,12 +72,15 @@ protected ServerSetup() {
 			}
 			
 			/* CONFIGS */
-			if (!SyncConfig.INCLUDE_LIST.isEmpty()) {
+			/* Either have the whitelist disabled *OR* enabled + check the include list */
```
Comment (rheimus) [inline src/main/java/com/superzanti/serversync/ServerSetup.java]: Is this intended to be a toggle between blacklist/whitelist?

**Comment by rheimus** [comment]
As this would break compatibility with 1.7.10 I cant merge it completely. I have however implemented your fix for Unix and the safety checks on JsonObjects.

Thank you for catching those, it is much appreciated.

**Comment by Lordmau5** [comment]
I should've pulled it to a secondary branch, you're right.
I might look into that later so I'm not breaking the 1.7.10 support

**Comment by rheimus** [comment]
I've created a branch that supports multiple build directories, currently has base SS 1.7.10 and 1.9.4 (untested as of yet, probably get around to it later).

Just a wrapper gradle project really that builds all the included projects in settings.gradle and drops them in a publish folder in the root dir.

Run builds from the root dir.

**Comment by rheimus** [comment]
Closing this as the multi-version branch tests successfully on mc-1.9.4

**Pull request closed without merging**